### PR TITLE
Add Notion 0.1.6 cask.

### PR DESCRIPTION
**Pull request by jesse-c**
#### Adding a new cask
- [x] Checked there aren’t open [pull requests](https://github.com/caskroom/homebrew-cask/pulls) for the same cask.
- [x] Checked there aren’t closed [issues](https://github.com/caskroom/homebrew-cask/issues) where that cask was already refused.
- [x] When naming the cask, followed the [token reference](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [x] Commit message includes cask’s name.
- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` left no offenses.
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.

### CODE REVIEW

File: Casks/notion.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask 'notion' do
+  version '0.1.6'
+  sha256 :no_check
```
Comment (adidalal) [inline Casks/notion.rb]: Needs a shasum set. `shasum -a 256 Notion-0.1.6.dmg` will get you the needed value

File: Casks/notion.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+  url 'https://www.notion.so/desktop/Notion-0.1.6.dmg'
```
Comment (adidalal) [inline Casks/notion.rb]: Use `"` and `#{version}` (string interpolation) instead of hardcoding in the version number

**Comment by jawshooah** [comment]
Thank you for the contribution. It needed some fixes, so they were made in ebf3454888614e4275103e90164c43d7ed4e8b83. Your contribution is still included (and still credited to you), with the appropriate modifications. Please feel free to ask about any of the changes.

**Comment by jesse-c** [comment]
Thanks @jawshooah!

**Pull request closed without merging**